src/heritedcode_api_functions.py

```diff
@@ -104,3 +104,22 @@ def get_content_information(checksum_type, checksum):
         status=response["status"],
     )
     return response_object
+
+
+def is_origin_present(origin_url):
+    """
+    Return a boolean flag if given origin is present in SWH archive.
+    """
+    assert (
+        type(origin_url) == str
+    ), f"origin_url: {origin_url!r} must be string, not {type(origin_url)!r}."
+
+    api = f"{BASE_SWH_API_URL}origin/search/{origin_url}?limit=1"
```

Comment:
Can you please make a function for getting SWH urls and also write a small doctest or test for that function ?

Reply:
I don't get what do you mean by function for Getting SWH urls. Do you mean a seperate function for getting all available origin urls present in SWH archive ??
Thanks :)